opencraft/settings.py

```diff
@@ -299,6 +299,11 @@
     'consumer': {'workers': 1, 'loglevel': logging.INFO},
 }
 
+# Instance Logs Server ########################################################
+
+INSTANCE_LOGS_SERVER_HOST = env('INSTANCE_LOGS_SERVER_HOST', default='logs.opencraft.com')
```

Comment:
@pcockwell, let's avoid specifying the actual hostname/domain name of one of our servers. You can use `FILEBAT_LOGSTASH_HOSTS[0]` as the default as the currently configured host list for that variable is the same server.

Reply:
Done in next commit

Reply:
I've got code in to address this, but it causes a runtime error in the tests because `FILEBEAT_LOGSTASH_HOSTS` doesn't have a value for tests. How can we ensure a value is set for this? Or should I just set the value of `INSTANCE_LOGS_SERVER_HOST` to `None` if there is no value for `FILEBEAT_LOGSTASH_HOSTS`?

Reply:
@pcockwell, I suggested the `FILEBEAT_LOGSTASH_HOSTS[0]` value only because we don't have to configure/add another variable to the `ansible-secrets` repository for this. You can also set `FILEBEAT_LOGSTASH_HOSTS` to a list with a single dummy value as long as the unit test(s) don't try to connect to that hostname. You can also directly set the default to `''` or `None` if the environment variable doesn't exist if you want to keep it simple.